bobtemplates/hooks.py

```diff
@@ -49,3 +51,22 @@ def get_plone_classifier_version(configurator, question, answer):
     configurator.variables['package.classifier_version'] = '.'.join(an_split[:2])
 
     return answer
+
+
+def get_deploy_scripts(configurator):
```

Comment:
I would rename this function: since it does not return anything it is not really a getter, right?
Maybe use `download_deploy_scripts` or `update_deploy_scripts` ?